tests/transport_test.go

```diff
@@ -78,13 +87,19 @@ var _ = Describe("Transport Tests", func() {
 		err = utils.WaitForPersistentVolumeClaimPhase(c, ns, v1.ClaimBound, pvc.Name)
 		Expect(err).NotTo(HaveOccurred(), "Error waiting for claim phase Bound")
 
-		By("Verifying PVC is not empty")
-		Expect(framework.VerifyPVCIsEmpty(f, pvc)).To(BeFalse())
+		if shouldSucceed {
+			By("Verifying PVC is not empty")
+			Expect(framework.VerifyPVCIsEmpty(f, pvc)).To(BeFalse())
+		} else {
+			By("Verifying PVC is empty")
+			Expect(framework.VerifyPVCIsEmpty(f, pvc)).ToNot(BeFalse())
+		}
 	}
 
 	DescribeTable("Transport Test Table", it,
-		Entry("should connect to http endpoint without credentials", httpNoAuthEp, false),
-		Entry("should connect to http endpoint with credentials", httpAuthEp, true))
+		Entry("should connect to http endpoint without credentials", httpNoAuthEp, false, "", "", true),
+		Entry("should connect to http endpoint with valid credentials", httpAuthEp, true, "", "", true),
```

Comment:
2nd Entry() should pass utils.AccessKeyValue and utils.SecretKeyValue

Reply:
done